### PR TITLE
Core24 169 moving consent up on the form

### PR DESCRIPTION
**Pull request by neb42**
Jira ticket: JIRA-CORE24-169

## What have you implemented?

* Add consent section to form
* Move consent questions there
* Move consent questions to start of file

## How to test it?
Look at the page

### CODE REVIEW

File: internal/views/individual_form.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 type IndividualForm struct {
 	*forms.Form
 	individual             *api.Individual
+	consentSection				 *forms.FormSection
```
Comment (dfigueroa87) [inline internal/views/individual_form.go:16]: Indentation is off

**Review by dfigueroa87 — COMMENTED** [review]
Looks good, just one comment

File: internal/views/individual_form.go [hunk 2]
```diff
@@ -255,6 +257,17 @@ func (f *IndividualForm) buildTitle(t locales.Translator) error {
...
 }
 
+func (f *IndividualForm) buildConsentSection(t locales.Translator) error {
+	f.consentSection = &forms.FormSection{
+		Title:       t("consent"),
+		Fields:      []forms.Field{},
+		Collapsible: true,
+		Collapsed:   false,
+	}
+	f.Form.Sections = append(f.Form.Sections, f.consentSection)
```
Comment (internetti) [inline internal/views/individual_form.go:267]: have you spoken to masa? ticket doesn't mention creating a new section

File: internal/views/individual_form.go (hunk 1 above)
Comment (neb42) [reply internal/views/individual_form.go:16]: Where? It looks ok to me

File: internal/views/individual_form.go (hunk 2 above)
Comment (neb42) [reply internal/views/individual_form.go:274]: I checked with Zineb (as her name was on the ticket), but you're right that this likely came from Masa so I'll double check with her.

**Pull request merged**